ui/components/ui/identicon/identicon.component.js

```diff
@@ -93,6 +97,10 @@ export default class Identicon extends Component {
         src={image}
         style={getStyles(diameter)}
         alt={alt}
+        onError={() => {
+          this.setState({ imageLoadingError: true });
+          this.forceUpdate();
```

Comment:
I absolutely loathe this but simply updating state doesn't cause a re-render

Reply:
TIL about forceUpdate